build.sbt

```diff
@@ -9,6 +11,12 @@ lazy val `course-management-tools` =
     .aggregate(cmta, cmtc, core, `functional-tests`, docs)
     .settings(commonSettings: _*)
     .settings(publish / skip := true)
+    .settings(inThisBuild(List(
```

Comment:
Sort of a total nit, but if you're using `inThisBuild` it doesn't actually need to be nested in here. It can live at the top level and also work fine.

Reply:
Makes sense; redundant stuff shouldn't be present. Removing it.

Reply:
Actually, the change you made isn't the same. So you'll notice now that your settings are _only_ applying to the aggregate, not all projects. So for example if you would look at the developers set for `cmta` you'll see there is none. So you'd need to either use `inThisBuild` at the top level, _or_ move the settings themselves into your `commonSettings` which then are given to each module.